htdocs/core/tpl/objectline_create.tpl.php

```diff
@@ -82,7 +82,7 @@
 	if ($this->situation_cycle_ref) {
 		print '<td class="linecolcycleref" align="right">' . $langs->trans('Progress') . '</td>';
 	}
-	if (! empty($usemargins))
+	if (! empty($usemargins) && !empty(($user->rights->margins->liretous)))
```

Comment:
$user->rights->margins->liretous is dedicated to see the margin done by other users.
If you add this test here, a user with permission to manage margin is no more able to enter the margin.
Test here should be
if (! empty($usemargins) && !empty(($user->rights->margins->creer)))